packages/default/scss/chat/_layout.scss

```diff
@@ -0,0 +1,383 @@
+    .k-timestamp {
+        font-size: smaller;
+        line-height: normal;
+        text-align: center;
+        align-self: stretch;
+    }
+
+
+    // Quick reply
+    .k-quick-replies {
+        margin-left: -$chat-message-list-padding-y;
+        margin-right: -$chat-message-list-padding-y;
+        padding-left: $chat-message-list-padding-y;
+        padding-right: $chat-message-list-padding-y;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        flex: 0 0 auto;
+        overflow-x: auto;
+        overflow-y: hidden;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    }
+    .k-quick-reply {
+        @include border-radius( 100px );
+        padding: $chat-quick-reply-padding-y $chat-quick-reply-padding-x;
+        border-width: 1px;
+        border-style: solid;
+        cursor: pointer;
+        user-select: none;
+        flex: 0 0 auto;
+    }
+    .k-quick-reply + .k-quick-reply {
+        margin-left: $chat-quick-reply-spacing;
+    }
+
+
+    // Message box
+    .k-message-box {
+        padding: $chat-message-box-padding-y $chat-message-box-padding-x;
+        border-width: 1px 0 0;
+        border-style: solid;
+        flex: 0 0 auto;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+
+
+        .k-input {
+            margin: 0;
+            padding: 0;
+            border: 0;
+            font: inherit;
+            background: none;
+            flex: 1 1 auto;
+        }
+    }
+
+}
+
+
+
+
+@include exports( "chat/cards/layout" ) {
+
+    .k-chat .k-card-deck {
+        margin-left: -$chat-message-list-padding-y;
+        margin-right: -$chat-message-list-padding-y;
+        padding-left: $chat-message-list-padding-y;
+        padding-right: $chat-message-list-padding-y;
+        overflow: hidden;
+        overflow-x: auto;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    }
+    .k-chat .k-card-deck .k-card {
+        flex-basis: 200px;
+    }
+}
+
+
+
+
+@include exports( "chat/typingindicator/layout" ) {
+
+    // Local variables
+    $chat-typing-indicator-dot-size: 8px !default;
+    $chat-typing-indicator-dot-spacing: 5px !default;
+
+    // Typing indicator
+    .k-typing-indicator {
+        padding: 0;
+        border-radius: 50px;
+        display: inline-flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+
+        span {
+            width: $chat-typing-indicator-dot-size;
+            height: $chat-typing-indicator-dot-size;
+            border-radius: 50%;
+            flex: 0 0 $chat-typing-indicator-dot-size;
+            background-color: currentColor;
+            opacity: 0.4;
+            @for $i from 1 through 3 {
+                &:nth-of-type(#{$i}) {
+                    animation: 1s blink infinite ($i * .3333s);
+                }
+            }
+        }
+        span + span {
+            margin-left: 5px;
+        }
+
+        @keyframes blink {
```

Comment:
I believe we should prefix animation identifiers in order to prevent clashes with animations in customer applications.